global.css

```diff
@@ -5226,7 +5226,7 @@ div[data-marker]:hover {
 
 .ogl-harvestOptions {
   display: flex;
-  justify-content: right;

```

Comment:
@Anonymes-User This was a conscious adjustment to improve UI. `gap: 0.5em` was added to add spacing.

Reply:
🤷  Fine